modules/minio/src/test/java/org/testcontainers/containers/MinIOContainerTest.java

```diff
@@ -0,0 +1,47 @@
+package org.testcontainers.containers;
+
+import io.minio.BucketExistsArgs;
+import io.minio.MakeBucketArgs;
+import io.minio.MinioClient;
+import io.minio.ObjectWriteResponse;
+import io.minio.StatObjectArgs;
+import io.minio.StatObjectResponse;
+import io.minio.UploadObjectArgs;
+import org.junit.Test;
+
+import java.net.URL;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MinIOContainerTest {
+
+    @Test
+    public void testBasicUsage() throws Exception {
+        MinIOContainer container = new MinIOContainer("minio/minio:latest");
+        container.start();
+
+        MinioClient minioClient = MinioClient
+            .builder()
+            .endpoint(container.getS3URL())
+            .credentials(container.getUserName(), container.getPassword())
+            .build();
+
+        minioClient.makeBucket(MakeBucketArgs.builder().bucket("test-bucket").region("us-west-2").build());
+
+        BucketExistsArgs existsArgs = BucketExistsArgs.builder().bucket("test-bucket").build();
+
+        assertThat(minioClient.bucketExists(existsArgs)).isTrue();
+
+        URL file = this.getClass().getResource("/object_to_upload.txt");
+        assertThat(file).isNotNull();
+        ObjectWriteResponse upload = minioClient.uploadObject(
```

Comment:
upload is not being used

Reply:
In general if an upload fails it throws an exception. If we get the upload object back then it was a success. I can make it more clear with a comment or remove the variable.

Reply:
removing the variable would be fine